### PR TITLE
Links Breeze documentation to new Breeze video

### LINKED ISSUE #8578: Describe contents of the production image for users
**Description**

Users need to know more details about production images - where the logs/dags etc. are, which variables might be set to trigger certain functionality.

Current documentation is more focused on building process rather than using it. This should be improved.

**Use case / motivation**

Users should be able to understand the content and usage of the production image.

Connected with #8542

### PR DESCRIPTION
**Pull request by potiuk**
Also Closes #8578

---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Target Github ISSUE in description if exists
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by potiuk** [comment]
@turbaszek -> once we merge this one we can send info to our workshop attendees about preparation.

**Comment by potiuk** [comment]
Nicely rendered Breeze doc with links to video here: https://github.com/PolideaInternal/airflow/blob/links-to-new-breeze-video/BREEZE.rst

File: BREEZE.rst [hunk 1]
```diff
@@ -66,17 +69,19 @@ Docker Community Edition
 
 Here is an example configuration with more than 200GB disk space for Docker:
 
-.. image:: images/disk_space_osx.png
-    :align: left
-    :alt: Disk space OSX
+
+    <div align="center">
```
Comment (mik-laj) [inline BREEZE.rst:74]: Screenshot 2020-07-11 at 18 13 38

File: BREEZE.rst [hunk 2]
```diff
@@ -388,8 +447,341 @@ Also - in case you run several different Breeze containers in parallel (from dif
...
+```apache/airflow`` repository. This Docker image (and Dockerfile) contains size-optimised Airflow
+installation with selected extras and dependencies. Its tag follows the pattern of
+``<BRANCH>-python<PYTHON_MAJOR_MINOR_VERSION>`` (for example, ``apache/airflow:master-python3.6``
+or ``apache/airflow:v1-10-test-python3.6``).
+
+However in many cases you want to add your own custom version of the image - with added apt dependencies,
+python dependencies, additional Airflow extras. Breeze's ``build-image`` command helps to build your own,
+customised variant of the image that contains everything you need.
+
+You can switch to building the production image by adding ``--production-image``flag to the ``build_image``
```
Comment (mik-laj) [inline BREEZE.rst]:
```suggestion
You can switch to building the production image by adding ``--production-image`` flag to the ``build_image``
```

File: BREEZE.rst [hunk 3]
```diff
@@ -15,35 +15,38 @@
...
-We called it *Airflow Breeze* as **It's a Breeze to develop Airflow**.
+We called it *Airflow Breeze* as **It's a Breeze to contribute to Airflow**.
 
 The advantages and disadvantages of using the Breeze environment vs. other ways of testing Airflow
 are described in `CONTRIBUTING.rst <CONTRIBUTING.rst#integration-test-development-environment>`_.
 
-Here is a short 10-minute video about Airflow Breeze (note that it shows an old version of Breeze. Some
-of the points in the video are not valid anymore. The video will be updated shortly with more up-to-date
-version):
+Watch the video below about Airflow Breeze. It explains motivation for Breeze
```
Comment (mik-laj) [inline BREEZE.rst]:
```suggestion
Watch the video below about Airflow Breeze. It explains the motivation for Breeze
```

File: BREEZE.rst [hunk 4]
```diff
@@ -66,17 +69,19 @@ Docker Community Edition
 
 Here is an example configuration with more than 200GB disk space for Docker:
 
-.. image:: images/disk_space_osx.png
-    :align: left
-    :alt: Disk space OSX
+
```
Comment (potiuk) [inline BREEZE.rst:73]:
```suggestion
.. raw:: html

```

**Comment by potiuk** [comment]
Good ones @mik-laj !

**Comment by potiuk** [comment]
@mik-laj  :)?

**Review by mik-laj — APPROVED** [review]

**Pull request merged**